server/src/store/Memory.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Memory store more for testing purpose than production use.
+ */
+export class MemoryStore {
+  constructor() {
+    this.store = {};
+    this.hasSocketAdapter = false;
+  }
+
+  async get(key, field) {
+    if (this.store[key] === undefined || this.store[key][field] === undefined) {
+      return '';
+    }
+    return this.store[key][field];
+  }
+
+  async getAll(key) {
+    if (this.store[key] === undefined) {
+      return [];
+    }
+
+    return this.store[key];
+  }
+
+  async set(key, field, value) {
+    if (this.store[key] === undefined) {
+      this.store[key] = {};
+    }
+    this.store[key][field] = value;
+  }
+
+  async del(key, field) {
+    if (this.store[key] === undefined || this.store[key][field] === undefined) {
+      return;
```

Comment:
[Redis returns 0](https://redis.io/commands/hdel) when the field doesn't exist. Do we want to match the Redis API in our stores, or define our own store API? I'm fine with either as long as it's consistent across stores.
```suggestion
      return 0;
```